doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -56,51 +56,40 @@ A name ``f`` can be declared as a coercion between a source user-defined class
 conditions holds:
 
  * ``D`` is a user-defined class, then the type of ``f`` must have the form
-   :g:`forall (x₁:A₁)..(xₙ:Aₙ)(y:C x₁..xₙ), D u₁..uₘ` where :math:`m`
+   :g:`forall (x₁:A₁)..(xₖ:Aₖ)(y:C v₁..vₙ), D u₁..uₘ` where :math:`m`
    is the number of parameters of ``D``.
  * ``D`` is ``Funclass``, then the type of ``f`` must have the form
-   :g:`forall (x₁:A₁)..(xₙ:Aₙ)(y:C x₁..xₙ)(x:A), B`.
+   :g:`forall (x₁:A₁)..(xₖ:Aₖ)(y:C v₁..vₙ)(x:A), B`.
  * ``D`` is ``Sortclass``, then the type of ``f`` must have the form
-   :g:`forall (x₁:A₁)..(xₙ:Aₙ)(y:C x₁..xₙ), s` with ``s`` a sort.
+   :g:`forall (x₁:A₁)..(xₖ:Aₖ)(y:C v₁..vₙ), s` with ``s`` a sort.
 
-We then write :g:`f : C >-> D`. The restriction on the type
-of coercions is called *the uniform inheritance condition*.
+We then write :g:`f : C >-> D`.
+
+.. _ambiguous-paths:
+
+When a coercion is added to the inheritance graph, new coercion paths
+which have the same classes as existing ones are ignored. The
+convertibility of the new and existing paths is checked and a warning
+is displayed when the check fails. When the :g:`x₁..xₖ` are exactly
+the :g:`v₁..vₙ` (in the same order), the coercion is said to satisfy
+the :gdef:`uniform inheritance condition`. When possible, we recommend
+using coercions that satisfy this condition in order to make the test
+for ambiguous coercion paths complete.
```

Comment:
I don't know what "complete" means here, OTOH I've never used this feature.

Reply:
Here it means: no false positive.

Reply:
Like in sound and complete, I eventually chose a wording "guarantees that no spurious warning will be generated", is that clearer?

Reply:
An improvement for sure.